### PR TITLE
feat: slider is improved!

### PR DESCRIPTION
**Pull request by JasperTan97**
I am not sure if I need a PR for this, but anyways here's the updated slider. It's more readable code, and the slider only updates when the update button is pressed. Also we can have as many connections as we want, previously it was 1

### CODE REVIEW

**Comment by JasperTan97** [comment]
The slider is for 
https://github.com/aica-technology/modulo-controllers/pull/167

File: Dockerfile [hunk 1]
```diff
@@ -16,6 +16,6 @@ RUN sudo apt-get update && sudo apt-get install -y \
   && sudo ldconfig \
```
Comment (domire8) [inline Dockerfile]: libxinerama1, libxcursor dev are not necessary IMO, also the line `sudo ldconfig` is not needed

**Review by domire8 — APPROVED** [review]
I'm not going to review in detail since this is not going into the AICA stack but try to remove as many commented lines as possible, also in the Dockerfile and the .devcontainer.

The rest works, well done! tested with the real robot

**Comment by JasperTan97** [comment]
Now we have non integer scale too wow

File: Dockerfile (hunk 1 above)
Comment (JasperTan97) [reply Dockerfile]: No worries, I just wanted you to know that this is here to use.

**Comment by JasperTan97** [comment]
Also, I'll leave this up for today, then merge if there's no more input

**Pull request merged**